windows/mdag_enable_disable/mdag_enable_disable.yml

```diff
@@ -0,0 +1,58 @@
+# Copyright 2023 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Description:
+#   This case is to test Microsoft Defender Application Guard (MDAG), which will opens the untrusted websites
+# in an isolated Hyper-V-enabled container. For more details, please refer to
+# https://learn.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-application-guard/md-app-guard-overview
+#
+- name: mdag_enable_disable
+  hosts: localhost
+  gather_facts: no
+  vars_files:
+    - "{{ testing_vars_file | default('../../vars/test.yml') }}"
+  tasks:
+    - name: "Test case block"
+      block:
+        - name: "Test setup"
+          include_tasks: ../setup/test_setup.yml
+
+        - name: "Skip test case"
+          include_tasks: ../../common/skip_test_case.yml
+          vars:
+            skip_msg: "Skip test case due to Hyper-V compatible hardware is 64bit processor, this guest OS is: {{ guest_os_ansible_architecture }}."
+            skip_reason: "Not Supported"
+          when: guest_os_ansible_architecture != "64-bit"
+
+        - name: "Skip test case"
+          include_tasks: ../../common/skip_test_case.yml
+          vars:
+            skip_msg: "Skip test case due to MDAG supported OS editions are 'Pro', 'Education' or 'Enterprise', this guest OS is: {{ guest_os_edition }}."
+            skip_reason: "Not Supported"
+          when:
+            - guest_os_edition | lower not in ['pro', 'enterprise', 'education']
+            - guest_os_product_type | lower == 'client'
+
+        - name: "Skip test case"
+          include_tasks: ../../common/skip_test_case.yml
+          vars:
+            skip_msg:
+              - "Skip test case due to MDAG supported OS version is 'Windows 10 version 1809 or later' and 'Windows 11'."
+              - "This guest OS major version: {{ guest_os_ansible_distribution_major_ver }}, build number: {{ guest_os_build_num }}."
+            skip_reason: "Not Supported"
+          when: >
+            (guest_os_ansible_distribution_major_ver | int == 10 and guest_os_build_num | int < 1809) or
```

Comment:
`guest_os_build_num` is the build number, `1809` is the update version.

Reply:
Yes. Thanks. Update it to build number 17763.